package.json

```diff
@@ -41,7 +41,7 @@
     "cra-test": "react-scripts test",
     "cra-eject": "react-scripts eject",
     "upgrade-dep": "yarn-or-npm upgrade-interactive --latest",
-    "lint": "eslint --fix .",
+    "lint": "eslint --fix --quiet ./src/**/*.js ./public/**/*.js",
```

Comment:
`./src/**/*.js ./public/**/*.js` to make sure we are not checking `build` and `node_modules` while later may is ignored by default 🤷‍♂ 

`--quite` just complains on errors and not warnings, [docs](https://eslint.org/docs/user-guide/command-line-interface#options)